tests/datasets/test_table_storage.py

```diff
@@ -34,3 +34,13 @@ def test_partitions_number() -> None:
 
     topic_spec = KafkaTopicSpec(Topic.REPLAYEVENTS)
     assert topic_spec.partitions_number == 1
+
+
+def test_topic_current_config_values() -> None:
```

Comment:
Can we add a test to exercise `except Exception:` path in  `snuba/subscriptions/scheduler_consumer.py`?

Reply:
added